temporal-testing-junit4/src/main/java/io/temporal/testing/TestWorkflowRule.java

```diff
@@ -291,6 +299,16 @@ public String getTaskQueue() {
     return taskQueue;
   }
 
+  /**
+   * Returns name of the task queue that test worker is polling.
+   *
+   * @return
+   */
+  public GetWorkflowExecutionHistoryResponse getWorkflowExecutionHistory(
+      GetWorkflowExecutionHistoryRequest request) {
```

Comment:
I would change the signature to 

getWorkflowExecutionHistory(Execution execution)
getWorkflowExecutionHistory(String workflowId)

As the creation of the request is duplicated all other the tests.